blocks/search-results/search-results.js

```diff
@@ -0,0 +1,128 @@
+import { createCardBlock } from '../card/card.js';
+
+let done = true;
+
+export default async function decorate(block) {
+  block.innerHTML = '';
+  const searchTerm = new URLSearchParams(window.location.search).get('q');
+  block.classList.add('card-container', 'three-columns');
+  if (searchTerm) {
+    const elementHeading = block.parentNode.parentNode.parentNode.querySelector('.section.search-page-heading');
+    const textNode = document.createElement('h1');
+    textNode.textContent = searchTerm;
+    elementHeading.append(textNode);
+    const mainDiv = document.createElement('div');
+    mainDiv.innerHTML = '<div class="results-loading-spinner"><div>';
+    const resultsDiv = document.createElement('div');
+    resultsDiv.classList.add('results-div');
+    mainDiv.append(resultsDiv);
+    block.append(mainDiv);
+    const tokenizedSearchWords = searchItems(searchTerm);
+    loadResults(tokenizedSearchWords, resultsDiv);
```

Comment:
IntelliJ complains about the missing await. Disabel the inspection for this line
```suggestion
    // noinspection ES6MissingAwait
    loadResults(tokenizedSearchWords, resultsDiv);
```

Reply:
Fixed